README.md

```diff
@@ -121,10 +121,20 @@ public protocol CellViewConfigType {
 }
 ```
 
-The another step is define the `DataSourceProvider` in charge of handle the data source and the `CellViewConfig` for each cell.
+Another step is to define the `DataSourceProvider` in charge of handling the data source and the `CellViewConfig` for each cell. 
+The `DataSourceProvider` exposes the `NumberOfExpandedParentCells` attribute which can be used to change the behavior of the accordion to only have a `single` item open at once or to have `multiple` items open at any given time. Take note that the default behavior is to have multiple items open.
```

Comment:
Lowercase `n` in `NumberOfExpandedParentCells` and remember to update this if you do decide to rename it.